portfolio/src/main/webapp/style.css

```diff
@@ -193,6 +347,38 @@ section.blue {
   clear: both;
 }
 
+
+@media screen and (max-width: 700px) {
+  .column.half {
+    -ms-flex: 100%;
```

Comment:
If your goal is browser compatibility, I think there are a few other prefixes you should add. AFAIK there is no IE app for mobile phones, for example. 

http://shouldiprefix.com/#flexbox

Reply:
okay, will add!